device/barefoot/x86_64-accton_wedge100bf_65x-r0/platform.json

```diff
@@ -3,392 +3,587 @@
         "Ethernet0": {
             "index": "1,1,1,1",
             "lanes": "0,1,2,3",
-            "alias_at_lanes": "Ethernet0,Ethernet1,Ethernet2,Ethernet3",
-            "breakout_modes": "1x100G[40G],2x50G,4x25G[10G]"
+            "breakout_modes": {
+                "1x100G[40G]": ["Ethernet0"],
+                "2x50G": ["Ethernet0", "Ethernet1"],
```

Comment:
"2x50G": ["Ethernet0", "Ethernet2"],

Reply:
We should fix the issue throughout the whole file for 2x50G mode.

Reply:
Modified as requested.